client-misk/src/test/kotlin/app/cash/backfila/client/misk/hibernate/RawParametersHibernateBackfillTest.kt

```diff
@@ -0,0 +1,83 @@
+package app.cash.backfila.client.misk.hibernate
+
+import app.cash.backfila.client.misk.Backfill
+import app.cash.backfila.client.misk.BackfillConfig
+import app.cash.backfila.client.misk.ClientMiskService
+import app.cash.backfila.client.misk.ClientMiskTestingModule
+import app.cash.backfila.client.misk.DbMenu
+import app.cash.backfila.client.misk.MenuQuery
+import app.cash.backfila.client.misk.UnshardedPartitionProvider
+import app.cash.backfila.client.misk.embedded.Backfila
+import app.cash.backfila.client.misk.embedded.createDryRun
+import app.cash.backfila.client.misk.testing.assertThat
+import app.cash.backfila.protos.service.Parameter
+import com.google.inject.Module
+import javax.inject.Inject
+import misk.hibernate.Id
+import misk.hibernate.Query
+import misk.hibernate.Transacter
+import misk.testing.MiskTestModule
+import okio.ByteString
+import okio.ByteString.Companion.encodeUtf8
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class RawParametersHibernateBackfillTest {
+  @Suppress("unused")
+  @MiskTestModule
+  val module: Module = ClientMiskTestingModule(false)
+
+  @Inject @ClientMiskService lateinit var transacter: Transacter
+  @Inject lateinit var backfila: Backfila
+
+  @Test fun testRawParameters() {
+    createSome(transacter)
+    val run = backfila.createDryRun<RawParametersHibernateTestBackfill>(
+        parameters = mapOf(
```

Comment:
are raw parameters still useful? Why not abandon them?!

Reply:
So if the need arises they can still use the client even if they need to do something weird with the parameters.